.eslintignore

```diff
@@ -3,3 +3,5 @@ dist
 lib
 storybook/public
 website/.next
+website/out
+website/hosting
```

Comment:
do these need to be added to .gitignore?

Reply:
yeah...ideally we won't commit these values....

Reply:
I've been thinking about it more and I'm wondering if we can take an approach a bit more like our Storybook instance. Could these files be pushed to a different branch rather than `master` in the same way that we publish storybook to the `gh-pages` branch?

Reply:
yes that's definitely possible as well

Reply:
so we'd monitor for changes to the website directory in some other branch? and just not keep website up to date with master?

Reply:
right, it would give us some leeway so that we can make changes on `master` that aren't necessarily ready to be deployed yet.

Reply:
yeah i buy that